realm/src/androidTest/java/io/realm/entities/AllTypes.java

```diff
@@ -23,6 +23,18 @@
 import io.realm.annotations.Required;
 
 public class AllTypes extends RealmObject {
+
+    public static final String CLASS_NAME = "AllTypes";
+    public static final String FIELD_STRING = "columnString";
```

Comment:
That would probably make sense yes. At this point we could probably trim our test entities a bit in any case since we have a lot of overlap between them.